pkg/logs/tailer.go

```diff
@@ -0,0 +1,199 @@
+package logs
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+
+	"github.com/logrusorgru/aurora"
+	log "github.com/sirupsen/logrus"
+
+	"github.com/stripe/stripe-cli/pkg/ansi"
+	"github.com/stripe/stripe-cli/pkg/stripeauth"
+	"github.com/stripe/stripe-cli/pkg/websocket"
+)
+
+const outputFormatJSON = "json"
+
+// LogFilters contains all of the potential user-provided filters for log tailing
+type LogFilters struct {
+	FilterIPAddress      string `json:"filter_ip_address,omitempty"`
+	FilterHTTPMethod     string `json:"filter_http_method,omitempty"`
+	FilterRequestPath    string `json:"filter_request_path,omitempty"`
+	FilterSource         string `json:"filter_source,omitempty"`
+	FilterStatusCode     string `json:"filter_status_code,omitempty"`
+	FilterStatusCodeType string `json:"filter_status_code_type,omitempty"`
+}
+
+// Config provides the cfguration of a Proxy
```

Comment:
`cfguration` => `configuration`?

Is this something we could share with the webhook listener? It seems similar.

Reply:
It is similar, but I don't like the idea of sharing the configs in case there are things that are special to each (in this case, the `LogFilters`). In a world where more specific configuration is added to each, this gets messy.